### ISSUE TITLE
tokenResponse is null after refreshing the page in msal-browse 2.0.0-beta.3

### ISSUE THREAD

**Issue by jrmcdona** [AzureAD/microsoft-authentication-library-for-js#1844]
>> Please follow the issue template below. Failure to do so will result in a delay in answering your question.

## Library
"@azure/msal-browser": "^2.0.0-beta.3",

## Framework
Angular 9

## Description
I have an Angular 9 app.   I had things working with Beta 2 but Beta 3 is causing challenges.

When I go incognito and sign in, I am getting a token. That works fine.    

If I refresh the page right after that, the tokenResponse is null and everything falls apart.

Also, getAllAccounts is filling up with bad data (see pic)

![image](https://user-images.githubusercontent.com/4327773/85960179-54945a00-b967-11ea-92a1-61b845178b95.png)

So how am I supposed to get the Bearer token in this case if token response is null?

This happens with both localStorage and sessionStorage.

Am I supposed to call acquireSilentToken, if so, how can I do that if I don't know who the current user is to get username and then getAllAccounts has bad data in it.

## Security
- [ ] Is this issue security related?

## Regression
- [ ] Did this behavior work before? 
Version:

## MSAL Configuration
 const msalConfig: msal.Configuration = {
          auth: {
            clientId: environment.msaClientId,
            authority: environment.msaIssuer,
            redirectUri: environment.oAuthRedirectUrl,
          },
          cache: {
            cacheLocation: "localStorage", // This configures where your cache will be stored
            storeAuthStateInCookie: false, // Set this to "true" if you are having issues on IE11 or Edge
          }
        };

## Reproduction steps
```js
// Provide relevant code snippets here.
// For Azure B2C issues, please include your policies.
```

## Browsers/Environment
- [ ] Chrome
- [ ] Firefox
- [ ] Edge
- [ ] Safari
- [ ] IE
- [ ] Other (Please add browser name here)

**Comment by jrmcdona** [comment]
@pkanher617 were you able to repro on your end?

**Comment by pkanher617** [comment]
@jrmcdona Unfortunately wasn't able to get this reproduced - I am not able to get the account objects with undefined values. Can you send a fiddler trace of this issue to my email? It is available on my profile page.

**Comment by tnorling** [comment]
@jrmcdona Apologies for the delay, the bug causing undefined accounts was identified and fixed in #1919
